config/sync/block.block.mainnavigation_2.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
 id: mainnavigation_2
```

Comment:
Can you raise a ticket to see about renaming this to something like sidebar_mainnavigation, at present it's a little ambiguous. The machine name should be generated from the block description.

Reply:
That will be my fault, I'll see about updating the machine names

Reply:
Not a fault, just something you pick up along the journey.